### PR TITLE
update installation doc to reflect current status

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@TomSweeneyRedHat @rhatdan @vrothberg PTAL. 

#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Updates installation doc

#### How to verify it

Try out the steps.

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

None

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by lsm5** [comment]
@TomSweeneyRedHat remind me please, so buildah.io gets the install page from this very install.md, yes?

**Comment by lsm5** [comment]
Does it make sense to move it entirely to buildah.io src like we have podman's install doc in podman.io?

**Comment by TomSweeneyRedHat** [comment]
@lsm5, yes, buildah.io links to the install.md here in the Buildah GitHub repository.  After this review completes, I'd be fine with moving it to bulidah.io like we did with Podman's, as long as we leave an install.md here with a link to the new location.

File: install.md [hunk 1]
```diff
@@ -69,15 +69,35 @@ sudo apt-get update
...
-provides packages for Debian 10. The Kubic packages for Debian Testing/Bullseye and Debian Unstable/Sid
-have been discontinued to avoid
-[conflicts](https://github.com/containers/buildah/issues/2797) with the official packages.
+If you would prefer newer (though not as well-tested) packages,
+the [Kubic project](https://build.opensuse.org/package/show/devel:kubic:libcontainers:stable/buildah)
+provides packages for Debian 10 and newer. The packages in Kubic project repos are more frequently
+updated than the one in Debian's official repositories, due to how Debian works.
+The build sources for the Kubic packages can be found [here](https://gitlab.com/rhcontainerbot/buildah/-/tree/debian/debian).
+
+CAUTION: On Debian 11 and newer, including Testing and Sid/Unstable, we highly recommend you use podman, skopeo and buildah ONLY from EITHER the Kubic repo
```
Comment (TomSweeneyRedHat) [inline install.md:78]: cap nits for the projects as in the podman variety.

File: install.md [hunk 2]
```diff
@@ -160,7 +183,16 @@ sudo apt-get -y update
...
 
-The [Kubic project](https://build.opensuse.org/package/show/devel:kubic:libcontainers:stable/buildah) provides packages for some older but supported Ubuntu versions (it should also work with direct derivatives like Pop!\_OS).
+If you would prefer newer (though not as well-tested) packages,
+the [Kubic project](https://build.opensuse.org/package/show/devel:kubic:libcontainers:stable/buildah)
+provides packages for active Ubuntu releases 18.04 and newer (it should also work with direct derivatives like Pop!\_OS).
+The packages in Kubic project repos are more frequently updated than the one in Ubuntu's official repositories, due to how Debian/Ubuntu works.
+Checkout the Kubic project page for a list of supported Ubuntu version and architecture combinations.
+The build sources for the Kubic packages can be found [here](https://gitlab.com/rhcontainerbot/buildah/-/tree/debian/debian).
+
+CAUTION: On Ubuntu 20.10 and newer, we highly recommend you use podman, skopeo and buildah ONLY from EITHER the Kubic repo
```
Comment (TomSweeneyRedHat) [inline install.md:193]: caps nit

**Comment by TomSweeneyRedHat** [comment]
LGTM other than a few cap nits.

**Comment by lsm5** [comment]
Done

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Comment by lsm5** [comment]
> @lsm5, yes, buildah.io links to the install.md here in the Buildah GitHub repository. After this review completes, I'd be fine with moving it to bulidah.io like we did with Podman's, as long as we leave an install.md here with a link to the new location.

@TomSweeneyRedHat That'd be great, thanks!

**Pull request merged**